amps/share-services/src/test/java/org/alfresco/slingshot/web/scripts/SlingshotContentGetTest.java

```diff
@@ -212,15 +213,58 @@ public void testRelativePath() throws Exception
         nodeService.deleteNode(rootFolder);
     }
 
-    public NodeRef createNodeWithTextContent(NodeRef parentNode, String nodeCmName, QName nodeType, String content)
+    public void testForcedAttachment() throws Exception
+    {
+        Repository repositoryHelper = (Repository) getServer().getApplicationContext().getBean("repositoryHelper");
+        NodeRef companyHome = repositoryHelper.getCompanyHome();
+
+        NodeRef rootFolder = createNode(companyHome, "rootFolder", ContentModel.TYPE_FOLDER);
+        NodeRef testhtml = createNodeWithTextContent(rootFolder,"testhtml", ContentModel.TYPE_CONTENT, "testhtml content", MimetypeMap.MIMETYPE_HTML);
```

Comment:
Careful about formatting, this method call and many of the following ones don't have proper spacing inbetween arguments. Just select the code that you added (e.g.: the whole unit test method) and run the formatter just on that piece of code.